sktime/forecasting/base/adapters/_tbats.py

```diff
@@ -89,21 +91,23 @@ def _fit(self, y, X=None, fh=None):
         """
         self._forecaster = self._instantiate_model()
         self._forecaster = self._forecaster.fit(y)
+        self._yname = y.name
 
         return self
 
-    def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+    def _predict(self, fh, X, return_pred_int=False, alpha=DEFAULT_ALPHA):
         """Forecast time series at future horizon.
 
         Parameters
         ----------
         fh : int, list, np.array or ForecastingHorizon
             Forecasting horizon
-        X : pd.DataFrame, optional (default=None)
-            Exogenous time series
+        X : (default=None)
+            NOT USED BY TBATS
```

Comment:
maybe say sth like "argument ignored but present for interface compatibility"?

Reply:
fixed this myself